packages/uicore/src/components/TableV2/TableV2.tsx

```diff
@@ -54,6 +54,7 @@ export interface TableProps<Data extends Record<string, any>> {
   autoResetExpanded?: boolean
   autoResetSortBy?: boolean
   autoResetPage?: boolean
+  initialState?: any
```

Comment:
please use the correct types. `any` is very dangerous as it disables typechecking

Reply:
Fixed https://github.com/harness/uicore/pull/1082/commits/041be85909d2dc1476523318c7200716d6081a70